assets/js/modules/adsense/datastore/constants.js

```diff
@@ -25,3 +25,7 @@ export const SITE_STATE_NEEDS_ATTENTION = 'NEEDS_ATTENTION';
 export const SITE_STATE_REQUIRES_REVIEW = 'REQUIRES_REVIEW';
 export const SITE_STATE_GETTING_READY = 'GETTING_READY';
 export const SITE_STATE_READY = 'READY';
```

Comment:
I would advise we remove these four constants, as they can become quite confusing in combination with our own states. To clarify, the states here are the possible API values, which can apply to the `state` field of accounts, clients, and sites. So _if_ we want to have constants for them, let's use the `STATE_` prefix like below. Or for even more clarity, we could use `API_STATE_`, just to clarify these are not our own account or site status values.